homeassistant/components/peco/sensor.py

```diff
@@ -46,11 +46,11 @@ async def async_setup_entry(
     websession = hass.data[DOMAIN][config_entry.entry_id]["websession"]
     conf: MappingProxyType[str, Any] = config_entry.data
 
-    async def async_update_data():
+    async def async_update_data() -> dict:
```

Comment:
Can this `dict` be narrowed down? Eg. `dict[str, int]`, `dict[str, int | str]` or `dict[str, Any]` depending on the actual return values of your pypi package.